cmd/benchmark/main.go

```diff
@@ -130,20 +128,6 @@ func main() {
 		defer pprof.StopCPUProfile()
 	}
 
-	if *traceFile != "" {
```

Comment:
Actually, this "tracing" code works fine. I was referring to the use of the `net/trace` package within Gorums. The `runtime/trace` package is different.

Reply:
Okay. I see. Do you mean that `grpc.EnableTracing` can be removed along with `trace.IsEnabled()` check line 195.

Reply:
Yes. But the `runtime/trace` stuff can stay